.NET/Microsoft.Recognizers.Text.DateTime/Extractors/IMergedExtractorConfiguration.cs

```diff
@@ -36,6 +36,8 @@ public interface IMergedExtractorConfiguration : IDateTimeOptionsConfiguration
 
         IEnumerable<Regex> TermFilterRegexes { get; }
 
+        IEnumerable<Regex> TasksSkipMergeRegexes { get; }
```

Comment:
Add a code commen here explaining the purpose of this and making it clear it's only used if the specific mode is enabled.